css/templates/e-commerce/page.css

```diff
@@ -118,24 +122,12 @@ dd:after {
   padding-right: 13px;
 }
 
-.commerce-select-wrapper::after {
```

Comment:
Remove down arrow on select elements. Now will be a lighter colour to signify an option, (as per designs).